app/lib/rocketchat.js

```diff
@@ -334,11 +334,11 @@ const RocketChat = {
 				const result = await this.login(params);
 				return resolve(result);
 			} catch (e) {
-				if (e.data?.error && (e.data.error === 'totp-required' || e.data.error === 'totp-invalid')) {
+				if (e.data.error && (e.data.error === 'totp-required' || e.data.error === 'totp-invalid')) {
 					const { details } = e.data;
 					try {
-						const code = await twoFactor({ method: details?.method || 'totp', invalid: e.data.error === 'totp-invalid' });
```

Comment:
Revert optional operators.

Reply:
Reverted the last commit. Sorry, I didn't know about the optional operator but now I know. Thanks for the review.